src/deploy-w3c-echidna.ts

```diff
@@ -93,9 +93,7 @@ export default async function main(inputs: Input, buildOutput: BuildResult) {
 }
 
 async function publish(input: Input, buildOutput: BuildResult) {
-	const outputDir = path.join(buildOutput.root, buildOutput.dir);
-	const file = path.relative(outputDir, buildOutput.file);
```

Comment:
`buildOutput.file` is already relative to `outputDir`. Getting it relative to `outputDir` added wrong parent directory to it.

(This stuff is hard to debug, as only way is to run it and see if it works)